pycbc/mchirp_area.py

```diff
@@ -0,0 +1,241 @@
+#!/usr/bin/env python
+#
+# Integration of the area laying in the different cbc regions
+# By A. Curiel Barroso
+# August 2019
+
+"""This script computes the area corresponding to different CBC on the m1 & m2
+plane when given a central mchirp value and uncertainty.
+It also includes a function that calculates the source frame when given the
+detector frame mass and redshift.
+"""
+
+from pycbc.conversions import mass2_from_mchirp_mass1 as m2mcm1
+from scipy.integrate import quad
+
+
+def src_mass_from_z_det_mass(z, del_z, mdet, del_mdet):
```

Comment:
Docstrings look fine, but you don't need to write 'this function' at the beginning of each one, it's clear that they are functions ;)